common/src/main/java/com/microsoft/identity/common/internal/broker/ipc/AccountManagerAddAccountStrategy.java

```diff
@@ -101,9 +105,16 @@ public Type getType() {
 
     @Override
     public boolean isSupportedByTargetedBroker(@NonNull final String targetedBrokerPackageName) {
```

Comment:
What does this code intending to do? In what scenario would this get invoked?

Reply:
This is a backup IPC mechanism between MSAL and Broker, which was introduced before we discovered ContentProvider.